fix: mutable dt alias from unique.data.table (#5960)
Rdatatable/data.table: NEWS.md

56. `split.data.table()` works for downstream methods that don't implement `DT[i]` form (i.e., requiring `DT[i, j]` form, like plain `data.frame`s), for example `sf`'s `[.sf`, [#5365](https://github.com/Rdatatable/data.table/issues/5365). Thanks @barryrowlingson for the report and @michaelchirico for the fix.

> 57. `unique()` no longer returns a mutable alias in the case when `nrows(x) <= 1`, [#5960](https://github.com/Rdatatable/data.table/pull/5960). Thanks to @brookslogan for the report and @dshemetov for the fix.

Reviewer: Please move to NEWS of `1.15.99`, ~line 24 of NEWS

> 57. `unique()` no longer returns a mutable alias in the case when `nrows(x) <= 1`, [#5960](https://github.com/Rdatatable/data.table/pull/5960). Thanks to @brookslogan for the report and @dshemetov for the fix.

Reviewer: ```suggestion
57. `unique()` no longer returns a mutable alias in the case when `nrows(x) <= 1`, [#5932](https://github.com/Rdatatable/data.table/pull/5932). Thanks to @brookslogan for the report and @dshemetov for the fix.
```

> 57. `unique()` no longer returns a mutable alias in the case when `nrows(x) <= 1`, [#5932](https://github.com/Rdatatable/data.table/pull/5932). Thanks to @brookslogan for the report and @dshemetov for the fix.

Reviewer 2: ```suggestion
57. `unique()` returns a copy the case when `nrows(x) <= 1` instead of a mutable alias, [#5932](https://github.com/Rdatatable/data.table/pull/5932). This is consistent with existing `unique()` behavior when the input has no duplicates but more than one row. Thanks to @brookslogan for the report and @dshemetov for the fix.
```

14. The options `datatable.print.class` and `datatable.print.keys` are now `TRUE` by default. They have been available since v1.9.8 (Nov 2016) and v1.11.0 (May 2018) respectively.

> 15. Thanks to @ssh352, Václav Tlapák, Cole Miller, András Svraka and Toby Dylan Hocking for reporting and bisecting a significant performance regression in dev. This was fixed before release thanks to a PR by Jan Gorecki, [#5463](https://github.com/Rdatatable/data.table/pull/5463).

Reviewer 2: ```suggestion
15. Thanks to @ssh352, Václav Tlapák, Cole Miller, András Svraka and Toby Dylan Hocking for reporting and bisecting a significant performance regression in dev. This was fixed before release thanks to a PR by Jan Gorecki, [#5463](https://github.com/Rdatatable/data.table/pull/5463). 
```
